ydb/core/statistics/aggregator/ut/ut_traverse_columnshard.cpp

```diff
@@ -12,14 +12,32 @@
 namespace NKikimr {
 namespace NStat {
 
-namespace {
+Y_UNIT_TEST_SUITE(TraverseColumnShard) {
+
+    Y_UNIT_TEST(TraverseColumnTable) {
+        TTestEnv env(1, 1);
+        auto init = [&] () {
+            CreateDatabase(env, "Database");
+            CreateColumnStoreTable(env, "Database", "Table", 10);
+        };
+        std::thread initThread(init);
 
+        auto& runtime = *env.GetServer().GetRuntime();
+        runtime.SimulateSleep(TDuration::Seconds(30));
+        initThread.join();
 
-} // namespace
+        auto pathId = ResolvePathId(runtime, "/Root/Database/Table");
 
-Y_UNIT_TEST_SUITE(TraverseColumnShard) {
+        runtime.SimulateSleep(TDuration::Seconds(30));
 
-    Y_UNIT_TEST(OneColumnTableTraversal) {
+        auto countMin = ExtractCountMin(runtime, pathId);
+
+        ui32 value = 1;
+        auto probe = countMin->Probe((const char *)&value, sizeof(value));
+        UNIT_ASSERT_VALUES_EQUAL(probe, 10);
+    }
+
+    Y_UNIT_TEST(TraverseColumnTableRebootTabletBeforeResolve) {
```

Comment:
```suggestion
    Y_UNIT_TEST(TraverseColumnTableRebootSaTabletBeforeResolve) {
```

Лучше везде указать, что это SaTablet перезапускаем.